content/blog/2021/06/introducing-the-psp-host-namespaces-policy.md

```diff
@@ -0,0 +1,156 @@
+  `8000-9000` in the Pod `.spec.containers[].ports[].hostPort` attributes.
+
+By default all settings are disallowed, so if you want to allow the
+usage of, say, the host network and the host PID namespace usage, the
+policy would need to have the following settings:
+
+```json
+{ "allow_host_network": true, "allow_host_pid": true }
+```
+
+### Running the policy
+
+The policy validates Pods at creation time.
+
+Let's run a couple of tests with [our CLI tool,
+`kwctl`](https://github.com/kubewarden/kwctl/).
+
+You will need at least `kwctl` version 0.1.7 to execute this examples.
+
+```console
+$ curl https://raw.githubusercontent.com/kubewarden/psp-host-namespaces/v0.0.7/test_data/pod_host_pid_enabled.json 2> /dev/null | \
+    kwctl run -r - registry://ghcr.io/kubewarden/policies/psp-host-namespaces:v0.0.7 | jq
+{
+  "uid": "1299d386-525b-4032-98ae-1949f69f9cfc",
+  "allowed": false,
+  "status": {
+    "message": "Pod has host PID enabled, but this is not allowed"
+  }
+}
+```
+
+Settings can also be provided as an empty string or an empty object in
+this case:
+
+```console
+$ curl https://raw.githubusercontent.com/kubewarden/psp-host-namespaces/v0.0.7/test_data/pod_host_pid_enabled.json 2> /dev/null | \
+    kwctl run --settings-json '' -r - registry://ghcr.io/kubewarden/policies/psp-host-namespaces:v0.0.7 | jq
+{
+  "uid": "1299d386-525b-4032-98ae-1949f69f9cfc",
+  "allowed": false,
+  "status": {
+    "message": "Pod has host PID enabled, but this is not allowed"
+  }
+}
+```
+
+```console
+$ curl https://raw.githubusercontent.com/kubewarden/psp-host-namespaces/v0.0.7/test_data/pod_host_pid_enabled.json | \
+    kwctl run --settings-json '{}' -r - registry://ghcr.io/kubewarden/policies/psp-host-namespaces:v0.0.7 | jq
+{
+  "uid": "1299d386-525b-4032-98ae-1949f69f9cfc",
+  "allowed": false,
+  "status": {
+    "message": "Pod has host PID enabled, but this is not allowed"
+  }
+}
+```
+
+As you can see it rejected our pod. This is because settings are
+defaulted to the most restrictive ones. Now, let's enable the PID
+namespace host usage:
+
+```console
+$ curl https://raw.githubusercontent.com/kubewarden/psp-host-namespaces/v0.0.7/test_data/pod_host_pid_enabled.json 2> /dev/null | \
+    kwctl run --settings-json '{ "allow_host_pid": true }' -r - registry://ghcr.io/kubewarden/policies/psp-host-namespaces:v0.0.7 | jq
+{
+  "uid": "1299d386-525b-4032-98ae-1949f69f9cfc",
+  "allowed": true
+}
+```
+
+And that works as expected!
+
+## Conclusion
+
+Stay tuned, because during the following weeks we will implement the
+[rest of the Pod Security Policies in separate, small Kubewarden
+policies](https://github.com/kubewarden/policy-hub/issues/32).
+
+You can use [this GitHub
+search](https://github.com/kubewarden/?q=psp&type=&language=&sort=) to
+find out what Kubewarden policies are implementing Pod Security
+Policies. You can also discover more policies in the [Policy
+Hub](https://hub.kubewarden.io/).
```

Comment:
I put the highlight only on policy hub.

Reply:
Do you mean dropping the GitHub search?

Reply:
Moved it to a new paragraph, is that better? Or should I drop the GitHub search?

Reply:
it's perfect now